eng/scripts/live-test-resource-cleanup.ps1

```diff
@@ -269,6 +269,17 @@ function FindOrCreateDeleteAfterTag {
     [object]$ResourceGroup
   )
 
+  if (!$ResourceGroup) {
```

Comment:
Do we have cases where this object is null?

Reply:
We don't, but I figured I'd add the handling anyway, in case we eventually turn on strict mode or something (not sure why I didn't enable that anyways)

Reply:
I suspect if we end up passing in null here it would be an error in the caller that might now be missed.

Reply:
Fair point (whoops, merged before I saw this), though this is a minor case and not the only code point that would be hit if the group object was null.